tests/queries/0_stateless/02022_storage_filelog_one_file.sh

```diff
@@ -14,23 +14,23 @@ user_files_path=$(clickhouse-client --query "select _path,_file from file('nonex
 
 for i in {1..20}
 do
-	echo $i, $i >> ${user_files_path}/a.txt
+	echo $i, $i >> ${user_files_path}/02022_storage_filelog_one_file.txt
 done
 
 ${CLICKHOUSE_CLIENT} --query "drop table if exists file_log;"
-${CLICKHOUSE_CLIENT} --query "create table file_log(k UInt8, v UInt8) engine=FileLog('${user_files_path}/a.txt', 'CSV');"
+${CLICKHOUSE_CLIENT} --query "create table file_log(k UInt8, v UInt8) engine=FileLog('${user_files_path}/02022_storage_filelog_one_file.txt', 'CSV');"
 
 ${CLICKHOUSE_CLIENT} --query "select * from file_log order by k;"
 
 for i in {100..120}
 do
-	echo $i, $i >> ${user_files_path}/a.txt
+	echo $i, $i >> ${user_files_path}/02022_storage_filelog_one_file.txt
 done
 
 ${CLICKHOUSE_CLIENT} --query "select * from file_log order by k;"
 
 # touch does not change file content, no event
-touch ${user_files_path}/a.txt
+touch ${user_files_path}/02022_storage_filelog_one_file.txt
```

Comment:
So the point of renaming was to allow these tests to be run in parallel with other tests, then need to remove `no-parallel` tag.
But as I remember flacky check for functional tests runs this test in parallel with itself, then this will not work, but adding `CURRENT_DATABASE` to file path should fix it.

Reply:
Also `CLICKHOUSE_TEST_ZOOKEEPER_PREFIX` can be used for the file name instead, since it is test name + database (or explicitly use `${CLICKHOUSE_TEST_NAME}_${CLICKHOUSE_DATABASE`)

Reply:
May be worth adding a second alias in `shell_config.sh` for `${CLICKHOUSE_TEST_NAME}_${CLICKHOUSE_DATABASE` like `TEST_UNIQUE_NAME` or any better name. Because this is very useful, but using `CLICKHOUSE_TEST_ZOOKEEPER_PREFIX` might be very confusing for a reader.

Reply:
Agree.

Reply:
So, I should add `CLICKHOUSE_TEST_UNIQUE_NAME` to `shell_config.h`, then update filename to it?

Reply:
Just one more line as here https://github.com/ClickHouse/ClickHouse/blob/master/tests/queries/shell_config.sh#L13
 but with name `CLICKHOUSE_TEST_UNIQUE_NAME` and replace `touch ${user_files_path}/02022_storage_filelog_one_file.txt`  with `touch ${user_files_path}/${CLICKHOUSE_TEST_UNIQUE_NAME}.txt` and remove `no_paralell` tag. Thanks for fixing!